src/data/checkfiles/default_checkfiles/validate-passthru-children.js

```diff
@@ -0,0 +1,59 @@
+import React from 'react'
+import { Link } from "@reach/router";
```

Comment:
Yesterday you mentioned something about `{ Link }`. Could it be due to the curly braces only wrapping a single element?

Seems to me the curly braces aren't needed.

Reply:
Yep, it's because of the curly braces around `{ Link }`. The thing is that the [reach router docs](https://reach.tech/router/) use the same import format: `import { Router, Link } from "@reach/router"`